src/autopredictor/bestscore.py

```diff
@@ -0,0 +1,16 @@
+def display_best_score(X):
+    """
+    This function Identifies the best score with respect to a specific scoring metric along with the corresponding model.
+    It gives the DataFrame output in a table format.
```

Comment:
Personally, I prefer the word 'returns' instead of gives. Just a personal opinion. :)

Reply:
I have made changes as per suggestion.